packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1976,11 +1984,13 @@ mixin WidgetInspectorService {
     final String groupName = parameters['groupName']!;
     final bool isSummaryTree = parameters['isSummaryTree'] == 'true';
     final bool withPreviews = parameters['withPreviews'] == 'true';
+    final bool fullDetails = parameters['abbreviated'] != 'true';
```

Comment:
Can we make these consistent? Either by naming the variable abbreviated and flipping the rest of the logic in this PR to have a default `abbreviated = false` parameter. Or by changing the parameter to 'fullDetails' to match the logic in this PR?

Reply:
Done, switched to changing the parameter to `fullDetails`. If it's omitted, we default to `true`.